changelogs/client_server/newsfragments/1552.clarification

```diff
@@ -0,0 +1 @@
+Added missing enumerated type for the m.reaction event.
```

Comment:
Please could you try to phrase this in a way that a reader of the spec will understand?

Reply:
```suggestion
Fix missing `type` property for the m.reaction event.
```